docs/module/vlan.md

```diff
@@ -148,6 +148,7 @@ To use a VLAN on a link, add **vlan** dictionary to a link or an interface on a
 
 * **access** -- the name of access VLAN configured on the link or interface
 * **trunk** -- a list or dictionary of VLANs configured on a trunk port -- see [](module-vlan-trunk-attributes) for more details.
+* **_allow_all** -- internal flag to mark a trunk for which all VLANs are allowed
```

Comment:
We're not documenting internal flags (for good reason -- we never promised the customers they can use them)

Reply:
removed